docs/snippets/all/archetypes/image_send_columns.rs

```diff
@@ -0,0 +1,57 @@
+use ndarray::{s, Array, ShapeBuilder};
+use rerun::Archetype as _;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let rec = rerun::RecordingStreamBuilder::new("rerun_example_image_send_columns").spawn()?;
+
+    // Timeline on which the images are distributed.
+    let times = (0..20).collect::<Vec<i64>>();
+
+    // Create a batch of images with a moving rectangle.
+    let width = 300;
+    let height = 200;
+    let mut images = Array::<u8, _>::zeros((times.len(), height, width, 3).f())
+        .as_standard_layout() // Make sure the data is laid out as we expect it.
+        .into_owned();
+    images.slice_mut(s![.., .., .., 2]).fill(255);
+    for &t in &times {
+        let t = t as usize;
+        images
+            .slice_mut(s![t, 50..150, (t * 10)..(t * 10 + 100), 1])
+            .fill(255);
+    }
+
+    // Log the ImageFormat and indicator once, as static.
+    let format = rerun::components::ImageFormat::from(rerun::datatypes::ImageFormat {
+        width: width as _,
+        height: height as _,
+        pixel_format: None,
+        color_model: Some(rerun::datatypes::ColorModel::RGB),
+        channel_datatype: Some(rerun::datatypes::ChannelDatatype::U8),
+    });
```

Comment:
I suggest we add a couple of helpers for a few common cases

```suggestion
    let format = rerun::components::ImageFormat::new_rgb8([width as _, height as _]);
```

Reply:
for some reason all the utilities ended up on the datatype, not the component :thinking: